docs/modules/cli/pages/jbang-init.adoc

```diff
@@ -65,6 +65,10 @@ Initialize a script.
 *--quiet*::
   jbang will be quiet, only print when error occurs.
 
+*-i*, *--ignore-template-defaults*=_<true|false>_::
```

Comment:
I don't see value in having this flag. why would you not want the defaults applied?